intercom-java/src/main/java/io/intercom/api/Company.java

```diff
@@ -257,19 +299,6 @@ public int getSessionCount() {
         return sessionCount;
     }
 
-    /**
-     * Deprecated. The Intercom API does not support changing the
-     * session value for a company. Calling this method has no
-     * effect.
-     *
-     * @param sessionCount this value is ignored
-     * @return the company object
-     */
-    @Deprecated
-    public Company setSessionCount(int sessionCount) {
-        return this;
-    }
```

Comment:
Removed this as it 
- prevented tests from passing 
- and doesn't seem to do anything of use and is probably overriding default behaviour causing and is causing issues such as https://github.com/intercom/intercom-java/issues/202 as well